manifests/configmap.yaml

```diff
@@ -11,8 +11,7 @@ data:
   debug_logging: "true"
   dns_name_format: '{cluster}.{team}.staging.{hostedzone}'
   docker_image: registry.opensource.zalan.do/acid/demospilo-10:1.3-p2
-  etcd_host: etcd-client.default.svc.cluster.local:2379
```

Comment:
It has to be an empty string, otherwise operator will use build-in default: https://github.com/zalando-incubator/postgres-operator/blob/eba23279c8bfce331c85fd5ca58df204568aceb6/pkg/util/config/config.go#L53

+ docker_image versions is p3